util/crc32c_ppc.c

```diff
@@ -0,0 +1,101 @@
+/* Copyright (C) 2017 International Business Machines Corp.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
```

Comment:
I am not sure of the answer to this, but I see that this is GPL licensed. Is introducing GPL code acceptable?